<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StereoPlotterForm</class>
 <widget class="QMainWindow" name="StereoPlotterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stereoplotter</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1062</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="featureListDockWidget">
   <property name="minimumSize">
    <size>
     <width>96</width>
     <height>121</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Feature list</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QTreeView" name="treeView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="featureEditorDockWidget">
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>223</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>250</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Feature editor</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Feature ID:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="featureIdLabel">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="nameEdit">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Feature type:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox_3">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Point</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Line</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Polygon</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Class:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboBox_4">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_12">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Number of points:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="noPointsLabel">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Perimeter:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="perimeterLabel">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Area:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="areaLabel">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="generalOptionsDockWidget">
   <property name="minimumSize">
    <size>
     <width>279</width>
     <height>197</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>250</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>General options</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Pair:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Stereo mode:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Anaglyph</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Polarized glasses</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Left anaglyph:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox_5">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Red</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Green</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Blue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cyan (Green + Blue)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Yellow (Red + Green)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Magenta (Red + Blue)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Right anaglyph:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBox_6">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="currentIndex">
        <number>3</number>
       </property>
       <item>
        <property name="text">
         <string>Red</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Green</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Blue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cyan (Green + Blue)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Yellow (Red + Green)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Magenta (Red + Green)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Stereo mode:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboBox_7">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Direct</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Reverse</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="featuresToolBar">
   <property name="windowTitle">
    <string>Feature tools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd"/>
   <addaction name="actionRemove"/>
   <addaction name="actionRemove_All"/>
   <addaction name="actionEnd"/>
  </widget>
  <widget class="QToolBar" name="filesToolBar">
   <property name="windowTitle">
    <string>File tools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad"/>
   <addaction name="actionSave"/>
   <addaction name="actionExport"/>
  </widget>
  <widget class="QToolBar" name="pointsToolBar">
   <property name="windowTitle">
    <string>Point tools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelect"/>
   <addaction name="actionInsert"/>
   <addaction name="actionEdit"/>
   <addaction name="actionRemove_Point"/>
  </widget>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/add.png</normaloff>:/icon/add.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="toolTip">
    <string>Add new feature</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/remove.png</normaloff>:/icon/remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove feature</string>
   </property>
  </action>
  <action name="actionRemove_All">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/removeall.png</normaloff>:/icon/removeall.png</iconset>
   </property>
   <property name="text">
    <string>Remove All</string>
   </property>
   <property name="toolTip">
    <string>Remove all features</string>
   </property>
  </action>
  <action name="actionEnd">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/featdone.png</normaloff>:/icon/featdone.png</iconset>
   </property>
   <property name="text">
    <string>End</string>
   </property>
   <property name="toolTip">
    <string>End feature</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/select.png</normaloff>:/icon/select.png</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="toolTip">
    <string>Select feature/point</string>
   </property>
  </action>
  <action name="actionInsert">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/ptadd.png</normaloff>:/icon/ptadd.png</iconset>
   </property>
   <property name="text">
    <string>Insert Mode</string>
   </property>
   <property name="toolTip">
    <string>Insert point mode</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/ptedit.png</normaloff>:/icon/ptedit.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="toolTip">
    <string>Edit point mode</string>
   </property>
  </action>
  <action name="actionRemove_Point">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/ptremove.png</normaloff>:/icon/ptremove.png</iconset>
   </property>
   <property name="text">
    <string>Remove Point</string>
   </property>
   <property name="toolTip">
    <string>Remove point</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/fileopen.png</normaloff>:/icon/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="toolTip">
    <string>Load features</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/filesave</normaloff>:/icon/filesave</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save features</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/report.png</normaloff>:/icon/report.png</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="toolTip">
    <string>Export features as...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
